features/importing/archivist.feature

```diff
@@ -225,3 +225,36 @@ Feature: Archivist bulk imports
       And I am logged in as "OpenDoors"
     When I go to the Open Doors tools page
     Then I should see "Update Redirect URL"
+
+  Scenario: Open Doors committee members can block an email address from having imports
+    Given I have an Open Doors committee member "OpenDoors"
+      And I have an archivist "archivist"
+      And the default ratings exist
+      And I am logged in as "OpenDoors"
+    When I go to the Open Doors tools page
+    Then I fill in "external_author_email" with "random@example.com"
```

Comment:
This and line 236 below it should be `And` -- you're telling the test to do something, so they go with the `When` above